### PR TITLE
"-w" / "--write" parameter is fixed in this PR

### PR DESCRIPTION
**Pull request by gkmngrgn**

### CODE REVIEW

File: unimport/unused.py [hunk 1]
```diff
@@ -1,5 +1,238 @@
...
+
+class ParseResult:
+    def __init__(self, commas, modules, newlines):
+        self.commas = commas
+        self.modules = modules
+        self.newlines = newlines
+
+
+class ImportTokenTransformer(TokenTransformer):
+    # You can find the original example from:
```
Comment (isidentical) [inline unimport/unused.py]: BRM license states you should keep copyright notice, not just a link 

https://github.com/isidentical/BRM/blob/dc54d39d824d8a8e9cd7fe666ad1156fea1775fe/LICENSE.txt#L9

**Comment by isidentical** [comment]
I dont have time to review actual implementation but the rest looks good to me (except a few catches). I'll try to find time for reviewing and testing actual implementation. Thanks so much for working on this @gkmngrgn

File: tests/samples/fix_multiple_problems_at_once_action.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import os
+import os.path
+import argparsefuck, sys
+import collections, collections.abc
+import importlib.util, importlib
+import multiprocessing.connection, multiprocessing.pool, multiprocessing.popen_fork
+import  http.server, http.client
+from a import b
+from c.d import e
+# from f.g import h, j.k
```
Comment (isidentical) [inline tests/samples/fix_multiple_problems_at_once_action.py]: why this is commented?

**Comment by isidentical** [comment]
The overall PR looks good, there are also a few nits which is not a problem for now. The biggest concern of mine is this PR lacks of tests, it has tests but this feature clearly requires more. Every impot statement in the `test.pym` from BRM examples should be tested individually like this;

```
input: from x import  y, z
names: y
output: from x import  y
```
```
input: import multiprocessing.workers, multiprocessing.pool
modules: multiprocessing.pool
output: multiprocessing.workers
```
etc. The current tests only verifies that all of them is removed but not the cases where one name is removed and the others stayed.

File: tests/samples/fix_multiple_problems_at_once_action.py (hunk 2 above)
Comment (gkmngrgn) [reply tests/samples/fix_multiple_problems_at_once_action.py]: @isidentical it's giving syntax error indeed but I keep this line to check if the overwrite process is keeping this line as expected.

```
======================================================================
ERROR: test_remove_unused_imports (tests.test_overwrite.OverwriteTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/gokmen/Workspace/alageek/unimport/tests/test_overwrite.py", line 14, in test_remove_unused_imports
    unused_imports = [
  File "/home/gokmen/Workspace/alageek/unimport/tests/test_overwrite.py", line 14, in <listcomp>
    unused_imports = [
  File "/home/gokmen/Workspace/alageek/unimport/unimport/unused.py", line 241, in get_unused
    dedect = DetectUnusedImport(source)
  File "/home/gokmen/Workspace/alageek/unimport/unimport/dedect.py", line 22, in __init__
    self.visit(ast.parse(source=source))
  File "/home/gokmen/.pyenv/versions/3.8.1/lib/python3.8/ast.py", line 47, in parse
    return compile(source, filename, mode, flags,
  File "<unknown>", line 10
    from f.g import h, j.k
                        ^
SyntaxError: invalid syntax
```

Comment (isidentical) [reply tests/samples/fix_multiple_problems_at_once_action.py]: Thanks

**Pull request merged**